kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -109,10 +166,22 @@ this will then trigger the successful processing of the traveller and put anothe
 }
 ```
 
+* To examine ProcessedTravellers topic and verify it was auto sent
```

Comment:
I would put this part before the `Send message`, so we can then send 2 different messages

Reply:
and so we could remove the `--from-beginning`

Reply:
Fixed @radtriste